lib/active_merchant/billing/gateways/ixopay.rb

```diff
@@ -14,7 +14,7 @@ class IxopayGateway < Gateway
       self.display_name = 'Ixopay'
 
       def initialize(options={})
-        requires!(options, :username, :password, :secret)
+        requires!(options, :username, :password, :secret, :api_key)
```

Comment:
We noticed that the ```api_key``` was missing as a requirement here, so went ahead and fixed that in this pull request.